src/tests/JIT/Regression/JitBlue/Runtime_113337/Runtime_113337.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Numerics;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+public class Program
```

Comment:
```suggestion
public class Runtime_113337
```

Reply:
Also, add the standard header (.NET Foundation)